lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -26,25 +26,28 @@ class CategoryRenderer {
   /**
    * @param {!ReportRenderer.AuditJSON} audit
    * @param {number} index
+   * @param {DocumentFragment=} providedTmpl
    * @return {!Element}
    */
-  renderAudit(audit, index) {
-    const tmpl = this.dom.cloneTemplate('#tmpl-lh-audit', this.templateContext);
+  renderAudit(audit, index, providedTmpl) {
```

Comment:
feels like this sorta becomes `populateAuditValues` instead, is that fair?

Reply:
essentially yeah.